setup.py

```diff
@@ -96,7 +96,10 @@ def read(fname):
     setup(
         name="err",
         version=VERSION,
-        packages=find_packages(src_root),
+        packages=find_packages(src_root, exclude=['tests',
+                                                  'tests.*',
```

Comment:
yeah I would prefer to leave it to the minimal set, it is clearer. We'll add them as necessary. thx !